### PR TITLE
BaiTap04 - TuNHT

### PR DESCRIPTION
**Pull request by at-tunguyen2**

### CODE REVIEW

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap1.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    func updateTime() -> String {
+        let date = Date()
+        let formatter = DateFormatter()
+        formatter.dateFormat = "hh:mm:ss dd-MM-yyyy"
+        let result = formatter.string(from: date)
+        return result
+    }
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        print("touches Began")
```
Comment (at-bienle) [inline BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap1.swift]: Add `super.TouchesBegan`

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap2.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        taoButton.layer.borderColor = UIColor(red: 0, green: 0, blue: 255, alpha: 1).cgColor
+        taoButton.layer.borderWidth = 1
+        taoButton.layer.cornerRadius = 10
+        
+        allButton.layer.borderColor = UIColor(red: 60, green: 30, blue: 90, alpha: 1).cgColor
+        allButton.layer.borderWidth = 1
+        allButton.layer.cornerRadius = 10
+    }
+    
+    @IBAction func chuoiButtonTouchUpInside(_ sender: UIButton) {
```
Comment (at-bienle) [inline BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap2.swift]: Add private for action, please check it in all your code

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap3.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    @IBOutlet private weak var thanhLabel: UILabel!
+    @IBOutlet private weak var vienLabel: UILabel!
+    @IBOutlet private weak var messiLabel: UILabel!
+    @IBOutlet private weak var cr7Label: UILabel!
+    @IBOutlet private weak var rooneyLabel: UILabel!
+    @IBOutlet private weak var pogbaLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (at-bienle) [inline BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap3.swift]: Remove line spacing

File: BaiTapOutlet&Action/BaiTapOutlet&Action/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//  ViewController.swift
+//  BaiTapOutlet&Action
+//
+//  Created by PCI0001 on 6/25/19.
+//  Copyright © 2019 PCI0001. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (at-bienle) [inline BaiTapOutlet&Action/BaiTapOutlet&Action/ViewController.swift]: Remove file if no need

**Comment by at-bienle** [comment]
@at-hainguyen3 Please follow this pull request!
Thank you!

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap1.swift (hunk 1 above)
Comment (at-tunguyen2) [reply BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap1.swift]: It has an error "Expression resolves to an unused function"

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap3.swift (hunk 3 above)
Comment (at-tunguyen2) [reply BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap3.swift]: I fixed it, thank you.

File: BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap2.swift (hunk 2 above)
Comment (at-tunguyen2) [reply BaiTapOutlet&Action/BaiTapOutlet&Action/BaiTap2.swift]: I fixed it, thank you.

**Pull request merged**